spring-integration-amqp/src/test/java/org/springframework/integration/amqp/support/RabbitTestContainer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.amqp.support;
+
+import java.time.Duration;
+
+import org.testcontainers.containers.RabbitMQContainer;
+
+/**
+ * Provides a static {@link RabbitMQContainer} that can be shared across test classes.
+ *
+ * @author Chris Bono
+ */
+public final class RabbitTestContainer {
```

Comment:
See `MySqlContainerTest`, `MongoDbContainerTest`, `MosquittoContainerTest` etc.
I really prefer to have them as interfaces.
And no any `IMAGE_CACHE`, please.

Reply:
No, we don't. The Harbor is used by default from its Docker environment on Bamboo.
It might be different on Jenkins and so, but I indeed sure that we have a proper mirroring on Bamboo.
I fact I removed those extra ENV vars from Spring AMQP CI plan to realign it with whatever works for me in Spring Integration plans.